src/defaults.js

```diff
@@ -1,6 +1,6 @@
 export default {
   level: `log`,
-  logger: console,
+  logger: (this && this['console']) || (window && window['console']),
```

Comment:
It's undefined until Developer Panel has been opened http://stackoverflow.com/questions/7742781/why-does-javascript-only-work-after-opening-developer-tools-in-ie-once